homework/hackyourtemperature/app.js

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import { API_KEY } from "./sources/keys.js";
+import fetch from "node-fetch";
+const app = express();
+
+app.get("/", (req, res) => {
+  res.send("hello from backend to frontend!");
+});
+
+app.use(express.json());
+app.post("/weather", async (req, res) => {
```

Comment:
For a better flow and readability of your code, try grouping similar concerns in the same areas in your code. For example, placing HTTP request code together. 

```suggestion

app.use(express.json());

app.get("/", (req, res) => {
  res.send("hello from backend to frontend!");
});

app.post("/weather", async (req, res) => {
```